didel/courses.py

```
@@ -121,6 +125,15 @@ def populate(self, soup, session):
         if about:
             self.about = about[0].get_text().strip()
 
+    def docs_and_links(self, path):
+        """
+        synchronize didel documents with user
+        """
+        if not (os.path.isdir(path)):
+            print("the path '" + path + "' dosen't exist")
```

raise an exception here instead of printing.